Documentation/Example/Package.swift

```diff
@@ -0,0 +1,40 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Kalimba",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Kalimba",
+            targets: ["Kalimba"]),
```

Comment:
Yep, I was inferring a package name from the example you preferred up front, the more important element in all this was that I was trying to match 'kalimba-benchmark' from your command-line examples. If you think it would make it more obvious as "KalimbaExample" or something, I'm totally good with changing it.

Reply:
Nah, this is fine! I have no problem with the library name.